### PR TITLE
Update Pupil Capture docs to v2.0

### PR DESCRIPTION
**Pull request by pfaion**
Update  section `Core / User Guide / Pupil Capture` to v2.0.

**Only merge once v2.0 has been released!**

Please review thoroughly, I had to rewrite quite a bit (including information that had been outdated since longer ago already).

### CODE REVIEW

File: src/core/software/pupil-capture.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ sudo usermod -a -G plugdev $USER
 ```
 
 ## Pupil Detection
-Pupil's algorithms automatically detect the participant's pupil. With the 3d detection and mapping mode, Pupil uses a 3d model of the eye(s) that constantly updates based on observations of the eye. This enables the system to compensate for movements of the headset - slippage. To build up an initial model, you can just look around your field of view.
+Pupil's algorithms automatically detect the participant's pupil. It runs two detection pipelines in parallel, the 2D and the 3D pupil detection. The 2D detection uses common computer vision technology to detect the pupil location in the camera image. The 3D detection uses a 3D model of the eye(s) that constantly updates based on observations of the eye. This enables the system to compensate for movements of the headset - slippage. To build up an initial model, it is recommended to briefly look around your field of view when putting on the headset.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:64]:
```suggestion
Pupil Core's algorithms automatically detect the participant's pupil. It runs two detection pipelines in parallel, the 2D and the 3D pupil detection. 

2D detection uses computer vision technology to detect the pupil location in the camera image. 

3D detection uses a 3D model of the eye(s) that updates based on observations of the eye. This enables the system to compensate for movements of the Pupil Core eye tracking headset on the participant's face (also known slippage). To build up an initial model, we recommend to look around your field of view when putting on the headset.
```

File: src/core/software/pupil-capture.md [hunk 2]
```diff
@@ -70,27 +70,29 @@ Pupil's algorithms automatically detect the participant's pupil. With the 3d det
...
 <v-icon large color="info">info_outline</v-icon>
 Keep in mind that pupil size values are defined in pixels and are therefore dependent on the resolution settings of your sensor.
 :::
 
 ## Calibration
-Pupil uses two cameras. One camera records a participant's eye movements - we call this the `eye camera`. Another camera records the subject's field of vision -we call this the `world camera`. In order to know what someone is looking at, we must find the parameters to a function that correlates these two streams of information.
+
+### Calibration Process
+
+Pupil Core headsets have two types of cameras attached. One camera records the subject's field of vision - we call this the `world camera`. Additionally there are one or more cameras recording the participant's eye movements - we call this the `eye cameras`. The data collected during the calibration period is used afterwards to correlate the world camera with the eye cameras.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:89]:
```suggestion
Pupil Core headsets have two types of cameras attached. One camera records the subject's field of vision - we call this the `world camera`. Additionally there are one or more cameras recording the participant's eye movements - we call these the `eye cameras`. The data collected during the calibration period is used afterwards to correlate the world camera with the eye cameras.
```

File: src/core/software/pupil-capture.md [hunk 3]
```diff
@@ -116,30 +119,31 @@ Make sure that the users pupil is properly detected and tracked. Make sure that
     </div>
 </div>
 
-### Calibration Methods
-Before starting calibration, ensure that eye(s) are robustly detected and that the headset is comfortable for the participant.
+### Choreographies
 
-#### Screen Marker Calibration
-This is the default method, and a quick way to get started. It is best suited for close range eye-tracking in a narrow field of view.
+All calibrations work in the way that specific markers are presented to the participant while the participant is instructed to look at the markers as closely as possible. The way in which markers are presented is called `Choreography`. Pupil offers different choreographies for common use cases.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:124]: Not all calibrations are marker based. 

```suggestion
All calibrations require a participant to look at a specific point in the real world or on screen. The way in which markers are presented is called a `Choreography`. Pupil Core provides different choreographies for common use cases.
```

File: src/core/software/pupil-capture.md [hunk 4]
```diff
@@ -116,30 +119,31 @@ Make sure that the users pupil is properly detected and tracked. Make sure that
...
+
+#### Screen Marker Calibration Choreography
+This is the default choreography, and a quick way to get started. It is best suited for close range eye-tracking in a narrow field of view.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:127]:
```suggestion
This is the default choreography, and a quick way to get started.
```

File: src/core/software/pupil-capture.md [hunk 5]
```diff
@@ -116,30 +119,31 @@ Make sure that the users pupil is properly detected and tracked. Make sure that
...
 
-1. Select `Screen Marker Calibration`
-1. Select your `Monitor` (if more than 1 monitor)
-1. Toggle `Use fullscreen` to use the entire extents of your monitor (recommended). You can adjust the scale of the pattern for a larger or smaller calibration target.
-1. Press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to start calibration.
-1. Follow the marker on the screen with your eyes. Try to keep your head still during calibration.
-1. The calibration window will close when calibration is complete.
+1. Select the `Screen Marker` choreography
+2. Select your `Monitor` (if more than 1 monitor)
+3. Toggle `Use fullscreen` to use the entire extents of your monitor (recommended). You can adjust the scale of the pattern for a larger or smaller calibration target.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:135]: Is `Use fullscreen` the default?

File: src/core/software/pupil-capture.md [hunk 6]
```diff
@@ -116,30 +119,31 @@ Make sure that the users pupil is properly detected and tracked. Make sure that
...
+5. Follow the marker on the screen with your eyes. Try to keep your head still during calibration.
+6. The calibration window will close when calibration is complete.
 
-In the `Advanced` sub-menu you can set the `sample duration` - the number of frames to sample the eye and marker position. You can also set parameters that are used to debug and detect the circular marker on the screen.
+If your participants have trouble following the markers, you can adjust the `Marker size` or `Sample duration` parameters.
 
 #### Calibration Marker
 
-If you're not going to be doing calibration on screen with the `Screen Marker Calibration` method, then you will need to download the Pupil's Circular Calibration Marker. This marker can be automatically detected by Pupil software in the world scene video. 
+If you're not going to do calibration on screen with the `Screen Marker` choreography, you will need to download the Pupil's Circular Calibration Marker. This marker can be automatically detected by Pupil software in the world scene video.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:144]:
```suggestion
If you're not going to do calibration on screen with the `Screen Marker` choreography, you will need to download the Pupil's Circular Calibration Marker. This marker can be automatically detected by Pupil software in the world video. 
```

File: src/core/software/pupil-capture.md [hunk 7]
```diff
@@ -160,56 +164,27 @@ Make sure to always use the **v0.4 marker design** for best detection performanc
 <a :href="$withBase('/pdfs/v0.4_marker.pdf')" alt="Pupil Labs calibration marker v4.0" target="_blank" rel="noopener">Download Pupil Labs Calibration Marker v0.4</a> to print or display on smartphone/tablet screen.
 
 
+#### Single Marker Calibration Choreography
+Calibrate using a single marker, either with a hand held physical markers or a digital marker on the screen. Gaze at the center of the marker and move your head in a spiral motion. You can also move your head in other patterns. This choreography enables you to quickly sample a wide range of gaze angles and cover a large range of your FOV.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:168]:
```suggestion
Calibrate using a single marker, either with a printed (physical) markers or a digital marker displayed on screen. Gaze at the center of the marker and move your head in a spiral motion. You can also move your head in other patterns. This choreography enables you to quickly sample a wide range of gaze angles and cover a large range of your FOV.
```

File: src/core/software/pupil-capture.md [hunk 8]
```diff
@@ -160,56 +164,27 @@ Make sure to always use the **v0.4 marker design** for best detection performanc
...
+1. Select `Single Marker` choreography
+2. Press `c` on your keyboard or click the blue circular `C` button on the left hand side of the world window to start calibration.
+3. Look at the center of the marker.
+4. Slowly move your head while gazing at the center of the marker. We have found that a spiral pattern is an efficient way to cover a large area of the FOV.
+5. Press the `C` button on your keyboard  or show the stop marker to stop calibrating.
 
 ::: tip
 <v-icon large color="info">info_outline</v-icon>
-If you're using a manual marker, make sure to select <code>Marker display mode > manual</code>. Also make sure that you do <strong>not</strong> display the marker on the screen when using a printed marker. Two markers visible to the world view at the same time will result in an inaccurate calibration.
+If you're using a physical marker, make sure to select <code>Physical Marker</code> in the <code>Marker display mode</code>. Also make sure that you do <strong>not</strong> display the marker on the screen at the same time when using a printed marker. Two markers visible to the world view at the same time will result in an inaccurate calibration.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:178]: This is the default now, not sure we need this tip anymore.

File: src/core/software/pupil-capture.md [hunk 9]
```diff
@@ -160,56 +164,27 @@ Make sure to always use the **v0.4 marker design** for best detection performanc
...
 
 ::: tip
 <v-icon large color="info">info_outline</v-icon>
 This paper introduces and evaluates this type of single marker calibration - <code>CalibMe: Fast and Unsupervised Eye Tracker Calibration for Gaze-Based Pervasive Human-Computer Interaction</code>
 :::
 
-#### Natural Features Calibration
-This method is for special situations and far distances. Usually not required.
+#### Natural Features Calibration Choreography
+This choregraphy is for special situations and far distances. Usually not required.
```
Comment (willpatera) [inline src/core/software/pupil-capture.md:187]:
```suggestion
This choregraphy is used only in special situations.
```

File: src/core/software/pupil-capture.md (hunk 8 above)
Comment (pfaion) [reply src/core/software/pupil-capture.md:182]: Agreed. I'll delete this.

File: src/core/software/pupil-capture.md (hunk 5 above)
Comment (pfaion) [reply src/core/software/pupil-capture.md:139]: It is.
Maybe we remove this point and add it below?
See suggestions there.

File: src/core/software/pupil-capture.md [hunk 10]
```diff
@@ -116,30 +123,31 @@ Make sure that the users pupil is properly detected and tracked. Make sure that
...
-1. The calibration window will close when calibration is complete.
+1. Select the `Screen Marker` choreography
+2. Select your `Monitor` (if more than 1 monitor)
+3. Toggle `Use fullscreen` to use the entire extents of your monitor (recommended). You can adjust the scale of the pattern for a larger or smaller calibration target.
+4. Press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to start calibration.
+5. Follow the marker on the screen with your eyes. Try to keep your head still during calibration.
+6. The calibration window will close when calibration is complete.
 
-In the `Advanced` sub-menu you can set the `sample duration` - the number of frames to sample the eye and marker position. You can also set parameters that are used to debug and detect the circular marker on the screen.
+If your participants have trouble following the markers, you can adjust the `Marker size` or `Sample duration` parameters.
```
Comment (pfaion) [inline src/core/software/pupil-capture.md:144]: @willpatera as I suggested in the comment above:

```suggestion
If your participants have trouble following the markers, you can adjust the `Marker size` or `Sample duration` parameters. We recommend to enable `Use fullscreen` for best marker visibility.
```

**Comment by pfaion** [comment]
I have modified this section again based on the new Gazer UI label `Gaze Mapping` which we decided on.

**Pull request merged**